beforeauto.mk.in: added support for overriding the configure time
defined CC/CXX/RANLIB/CPP variables so that a couple pkg tools could be
built which generate code during the cross compile build.

http://www.omniorb-support.com/pipermail/omniorb-list/2013-September/031473.html

Signed-off-by: Matt Weber <mlweber1@rockwellcollins.com>

--- omniorb-4.1.6.orig/mk/beforeauto.mk.in	2013-09-12 12:49:55.668708148 -0500
+++ omniorb-4.1.6/mk/beforeauto.mk.in	2013-09-12 12:51:22.707928579 -0500
@@ -672,15 +672,15 @@
 #
 
 AR              = ar cq
-CC              = @CC@
-CXX             = @CXX@
-RANLIB		= @RANLIB@
+CC		?= @CC@
+CXX		?= @CXX@
+RANLIB		?= @RANLIB@
 MKDIRHIER	= $(BASE_OMNI_TREE)/bin/scripts/omkdirhier
 INSTLIBFLAGS	= -m 0644
 INSTEXEFLAGS	= -m 0755
 CP		= cp
 MV		= mv -f
-CPP		= @CPP@
+CPP		?= @CPP@
 OMKDEPEND	= $(TOOLBINDIR)/omkdepend
 RMDIRHIER	= $(RM) -rf
 
